app/bower_components/angular-material/angular-material-mocks.js

```
@@ -0,0 +1,186 @@
+ *
+ * Developers interested in running their own custom unit tests WITH angular-material.js loaded...
+ * must also include this *mocks* file. Similar to `angular-mocks.js`, `angular-material-mocks.js`
+ * will override and disable specific AngularJS Material performance settings:
+ *
+ *  - Disabled Theme CSS rule generations
+ *  - Forces $mdAria.expectWithText() to be synchronous
+ *  - Mocks $$rAF.throttle()
+ *  - Captures flush exceptions from $$rAF
+ *
+ */
+(function(window, angular, undefined) {
```

Biome (1.9.4)

[error] Do not shadow the global "undefined" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)